packages/web-react/src/components/Dropdown/README.md

```diff
@@ -15,6 +15,22 @@ import { Dropdown, Button } from '@lmc-eu/spirit-web-react/components';
 </Dropdown>
 ```
 
+### Dropdown with Item
+
+Enhance your DropdownPopover by incorporating the versatile [Item] component.
```

Comment:
```suggestion
Enhance your DropdownPopover by incorporating the versatile [Item][item] component.
```
Are you sure that this is working?

Reply:
For me it is working:
<img width="619" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/26870554/4619a865-4b6e-41a4-b57f-0c0b0735c0a9">

I copied that from another readme 😁

Reply:
fyi, I think this doesn't work in the storybook readme embed. But it probably should be fixed there.

Reply:
hmm, i opened [Button readme](https://spirit-design-system-storybook.netlify.app/?path=/docs/components-button--overview) in storybook and there are broken links:
![image](https://github.com/lmc-eu/spirit-design-system/assets/26870554/4892139d-d97a-44b9-ae1e-f8e3bdf59f3b)

Reply:
i'll fix it and add links